### PR TITLE
Game board

### PR DESCRIPTION
**Pull request by annagaiel**
Do not create a new game to see the chessboard.
To see the chessboard go to http://localhost:3030/games/show
Run rake db:migrate to add the name field on the game db 
To see game form, click new game. 
User is not yet connected to the game in this chessboard show page.

### CODE REVIEW

**Comment by ghost** [comment]
Ah crap, I was in the wrong branch. I was wondering why I didn't see a lot of the things you had talked about Anna. Let me go into this branch and add my stuff in before we merge.

**Comment by ghost** [comment]
Ok, check this out. Don't forget to run rake db:migrate so that your local matches this remote. 
Anna, let me know what you think. I rerouted the nav bar to games/index.html.erb
We are now listing all available games (which is nil) and we can create a new game!

**Comment by ghost** [comment]
oops sorry closed on accident

**Comment by ghost** [comment]
Oh and don't forget to run bundle install. We have simple form gem on here, and this will make your local match this remote.

File: app/models/game.rb [hunk 1]
```diff
@@ -4,6 +4,16 @@ class Game < ActiveRecord::Base
 
 	after_create :populate_board!
 
+
+	def index
+		@list_all_games=Game.all
+	end
```
Comment (washburnad) [inline app/models/game.rb:11]: we typically just call this @games. At a later point we may want to list only certain kinds of games and we can deal with   naming then.

File: app/views/games/show.html.erb [hunk 2]
```diff
@@ -1 +1,22 @@
-<h1>Game's show page</h1>
+<div class="row">
+  <div class="medium-4 medium-offset-4 columns">
+  	<h1>Have Fun</h1>
+			<table class="chessboard" >
+				<% (0..7).each do |row| %>
+					<tr>
+					<% (0..7).each do |col|  %>
+						<% if ((row % 2 == 0) && (col % 2 != 0)) || ((row % 2 != 0) && (col % 2 == 0)) %>
```
Comment (washburnad) [inline app/views/games/show.html.erb]: We like to see very little 'business logic' in the views.  Could you create a helper for this?

File: app/models/game.rb (hunk 1 above)
Comment (ghost) [inline app/models/game.rb:11]: Thanks Aaron, I'll get to changing the name today!

**Comment by annagaiel** [comment]
I checked this with david.

**Pull request merged**